File: src/063. Powerful digit counts/063.lua
local count = 0
for n = 1, math.huge do
  start = math.ceil((10 ^ (n - 1)) ^ (1.0 / n))
  count = count + 9 - start + 1
  if start > 9 then
    break
  end
end
print(count)

File: src/063. Powerful digit counts/063.go
package main

import (
	"fmt"
	"math"
)

func main() {
	count := 0
	for n := 1; ; n++ {
		start := int(math.Ceil(math.Pow(math.Pow10(n - 1), (1.0 / float64(n)))))
		count += 9 - start + 1
		if start > 9 {
			break
		}
	}
	fmt.Println(count)
}
